copylot/gui/_qt/custom_widgets/photom_live_window.py

```diff
@@ -0,0 +1,382 @@
+from PyQt5.QtGui import QPainter, QPen, QImage, QColor, QPainterPath, QBrush
+from PyQt5.QtWidgets import (
+    QMainWindow,
+    QGraphicsScene,
+    QGraphicsSimpleTextItem,
+    QGraphicsItem,
+    QGraphicsView,
+    QWidget,
+    QApplication,
+    QGraphicsPathItem,
+    QGraphicsEllipseItem,
+    QGraphicsRectItem,
+)
+from PyQt5.QtCore import Qt, QEvent, QPoint, QRect, QPointF
+from PyQt5 import QtGui
+from copylot.gui._qt.photom_control.utils.update_dac import signal_to_dac
+
+"""
+This script creates a LiveViewWindow to control laser spot when overlaid with camera display.
+"""
+
+class LiveViewWindow(QMainWindow):
```

Comment:
QMainWindow is not really a QWidget as a module, feel free  to make a separate subpackage for custom_windows @edyoshikun